docs/user/package-management/repo-management.md

```diff
@@ -7,6 +7,23 @@ summary: Repository Management
 
 While Solus currently only provides two repositories, stable (shannon) and unstable, our package manager does support actively using multiple repositories on your system. This is useful if you are using a vendor-provided repository.
 
+## Official Solus Repositories
+
+The official Solus repositories are:
+
+- Stable (shannon)
```

Comment:
I wonder how this would look as a table instead of nested lists.

Reply:
Some screenshots from a 1080 display:

*Bullets*
![Capture](https://github.com/getsolus/help-center-docs/assets/23007135/c3c01f51-7eef-480c-a069-7505015c917b)

*Table, note: the url does not get line-broken when you copy it*
![Capture2](https://github.com/getsolus/help-center-docs/assets/23007135/ec8e71f0-8c66-4726-b648-b5e1d998a6e9)

Reply:
I have no strong preference here